docs/source/resources/scyllaclusters/nodeoperations/volume-expansion.md

```diff
@@ -0,0 +1,69 @@
+# Resizing storage in ScyllaCluster
+
+
+::::{caution}
+The StorageClass used for your PersistentVolumeClaims must support volume expansion. To verify if your StorageClass supports resizing, run:
+
+```bash
+kubectl get storageclass
+```
+
+Check the `allowVolumeExpansion` column. It should be set to `true` for StorageClass used by ScyllaCluster.
+::::
+
+
+Let's use the following ScyllaCluster as an example.
+```bash
+$ kubectl get scyllacluster
+NAME     READY   MEMBERS   RACKS   AVAILABLE   PROGRESSING   DEGRADED   AGE
+scylla   3       3         1       True        False         False      62m
+```
+
+## Increasing the storage size
+
+1. Since we are going to delete the ScyllaCluster object, make sure to have a copy of the existing one. You will need it later.
```

Comment:
How about adding an explicit command that can be used to dump the current state of the ScyllaCluster?

Reply:
But people reading it will have to do it anyway, and it might not be obvious to them how to achieve that, no? Maybe something like below could cover this?

```bash
kubectl get scyllacluster scylla -o json | jq 'del(
    .metadata.creationTimestamp,
    .metadata.generation,
    .metadata.uid,
    .metadata.resourceVersion,
    .status
)' | yq -P > scyllaClusterDefinition.yaml
```

Reply:
It's obvious if you're using gitops, where you keep either helm values or scylla cluster spec in repository. Then it's enough to trigger resync to recreate the object.

However not everybody does, so maybe it's better to include this step. 
Added.

Reply:
@rzetelskik :+1: You're right, the following works as well and doesn't require jq.
```shell
kubectl get scyllacluster scylla -o yaml | yq 'del(
    .metadata.creationTimestamp,
    .metadata.generation,
    .metadata.uid,
    .metadata.resourceVersion,
    .status
)' > scyllaClusterDefinition.yaml
```

Reply:
good idea, done